unattended.sh

```diff
@@ -1,186 +1,803 @@
+
+	update_firewall_config
+
+	enable_new_services
+
+	# install Centreon SELinux packages first (as getenforce is still at 0)
+	$PKG_MGR -q install -y ${CENTREON_SELINUX_PACKAGES[@]} --enablerepo="$CENTREON_REPO"
+	if [ $? -ne 0 ]; then
+		log "ERROR" "Could not install Centreon SELinux packages"
+	else
+		log "INFO" "Centreon SELinux rules are installed. Please consult the documentation https://docs.centreon.com/$CENTREON_MAJOR_VERSION/en/administration/secure-platform.html for more details."
+	fi
+
+	#then change the SELinux mode
+	set_runtime_selinux_mode $selinux_mode
+
+	set_selinux_config $selinux_mode
+}
+#========= end of function update_after_installation()
+
+#####################################################
+################ MAIN SCRIPT EXECUTION ##############
+
+## Process the provided arguments in line
+case "$1" in
+
+upgrade)
+	operation="upgrade"
+	parse_subcommand_options "$@"
+	;;
+
+install)
+	operation="install"
+	parse_subcommand_options "$@"
+	;;
+
+*)
+	log "WARN" "No provided operation : default value [$operation] will be used"
+	#usage
+	operation="install"
+	parse_subcommand_options "$@"
+	;;
+
+esac
+
+## Display all configured parameters
+log "INFO" "Start to execute operation [$operation] with following configuration parameters:"
+log "INFO" " topology   : \t[$topology]"
+log "INFO" " version    : \t[$version]"
+log "INFO" " repository : \t[$repo]"
+
+log "WARN" "It will start in [$default_timeout_in_sec] seconds. If you don't want to wait, press any key to continue or Ctrl-C to exit"
+pause "" $default_timeout_in_sec
+
+##
+# Analyze system and set the variables
+##
+set_required_prerequisite
+
+##
+# Check if systemd is present
+##
+is_systemd_present
+
+## Start to execute
+case $operation in
+
+install)
+	setup_before_installation
+	case $topology in
+	central)
+		CENTREON_SELINUX_PACKAGES=(centreon-common-selinux centreon-web-selinux centreon-broker-selinux centreon-engine-selinux centreon-gorgoned-selinux centreon-plugins-selinux)
+		install_central
+		CENTREON_DOC_URL="https://docs.centreon.com/$CENTREON_MAJOR_VERSION/en/installation/web-and-post-installation.html"
+		;;
+
+	poller)
+		CENTREON_SELINUX_PACKAGES=(centreon-common-selinux centreon-broker-selinux centreon-engine-selinux centreon-gorgoned-selinux centreon-plugins-selinux)
+		install_poller
+		CENTREON_DOC_URL="https://docs.centreon.com/$CENTREON_MAJOR_VERSION/en/monitoring/monitoring-servers/add-a-poller-to-configuration.html"
+		;;
+	esac
+
+	update_after_installation
+
+	if [ "$topology" == "central" ]; then
+		if [ "$wizard_autoplay" == "true" ]; then
+			play_install_wizard
+		fi
+	fi
+
+	log "INFO" "Centreon [$topology] successfully installed !"
+
+	if [ "$topology" == "central" ] && [ "$wizard_autoplay" == "true" ]; then
+		log "INFO" "Log in to Centreon web interface via the URL: http://$central_ip/centreon"
+	else
+		log "INFO" "Follow the steps described in Centreon documentation: $CENTREON_DOC_URL"
```

Comment:
could be merged on ligne 769 using the else

Reply:
not if we want these messages to be displayed after the log that is between both blocks